v3/setup/klam-ssh.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+AZ=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+REGION=${AZ::-1}
+ROLE_NAME="$(etcdctl get /klam-ssh/ROLE_NAME)"
+ENCRYPTION_ID="$(etcdctl get /klam-ssh/ENCRYPTION_ID)"
+ENCRYPTION_KEY="$(etcdctl get /klam-ssh/ENCRYPTION_KEY)"
+KEY_LOCATION_PREFIX="$(etcdctl get /klam-ssh/KEY_LOCATION_PREFIX)"
+IMAGE="$(etcdctl get /images/klam-ssh)"
+
+
+if [[ $REGION == "eu-west-1" ]]; then
+    KEY_LOCATION="-ew1"
+elif [[ $REGION == "ap-northeast-1" ]]; then
+    KEY_LOCATION="-an1"
+elif [[ $REGION == "us-east-1" ]]; then
+    KEY_LOCATION="-ue1"
+elif [[ $REGION == "us-west-1" ]]; then
+    KEY_LOCATION="-uw1"
+elif [[ $REGION == "us-west-2" ]]; then
+    KEY_LOCATION="-uw2"
+else
+    echo "An incorrect region value specified"
+    exit 1
+fi
+
+# create nsswitch.conf
+echo "passwd:     files usrfiles ato" > /home/core/nsswitch.conf
+echo "shadow:     files usrfiles ato" >> /home/core/nsswitch.conf
+echo "group:      files usrfiles ato" >> /home/core/nsswitch.conf
+echo -e "\n" >> /home/core/nsswitch.conf
+echo "hosts:      files usrfiles dns" >> /home/core/nsswitch.conf
+echo "networks:   files usrfiles dns" >> /home/core/nsswitch.conf
+echo -e "\n" >> /home/core/nsswitch.conf
+echo "services:   files usrfiles" >> /home/core/nsswitch.conf
+echo "protocols:  files usrfiles" >> /home/core/nsswitch.conf
+echo "rpc:        files usrfiles" >> /home/core/nsswitch.conf
+echo -e "\n" >> /home/core/nsswitch.conf
+echo "ethers:     files" >> /home/core/nsswitch.conf
+echo "netmasks:   files" >> /home/core/nsswitch.conf
+echo "netgroup:   nisplus" >> /home/core/nsswitch.conf
+echo "bootparams: files" >> /home/core/nsswitch.conf
+echo "automount:  files nisplus" >> /home/core/nsswitch.conf
+echo "aliases:    files nisplus" >> /home/core/nsswitch.conf
```

Comment:
Another stylistic nitpick, but can't we do something like:
```
cat <<EOT >> /home/core/nsswitch.conf
passwd: files usrfiles ato
shadow: files usrfiles ato
...
EOT
```

Just makes things easier to read / cleaner.